README.md

```diff
@@ -114,7 +137,7 @@ You can launch the following commands to download and test the latest distributi
 $> curl -L $(curl -s -L -X GET https://api.github.com/repos/dynawo/dynawo/releases/latest | grep "Dynawo_Linux" | grep url | cut -d '"' -f 4) -o Dynawo_Linux_latest.zip
 $> unzip Dynawo_Linux_latest.zip
 $> cd dynawo
-$> ./dynawo.sh jobs-with-curves Dynawo_Linux_latest/testcases/IEEE14/IEEE14_SyntaxExamples/IEEE14_ModelicaModel/IEEE14.jobs
+$> ./dynawo.sh jobs-with-curves sources/examples/DynaWaltz/IEEE14/IEEE14_GeneratorDisconnections/IEEE14.jobs
```

Comment:
testcases was just a copy of the nrt. Maybe we should put the examples instead and rename testcases in examples ? And after some exchanges with Matthieu to include the sources is not really mandatory to distribute the binaries as long as we are open source so we could remove this folder (for a lighter tar.gz)

Reply:
I'm completely in phase with this.

At the moment, I suggest to keep pointing to the examples directory in the sources because I want people to use the examples and the test cases are not yet the examples for this version.
But for the next release, the test cases should be the examples and not the nrt and we will then directly point to the testcases (renamed examples perhaps) directory.